### ISSUE TITLE
Create code representation for basis function regressors

### ISSUE THREAD

**Issue by wilsonrljr** [wilsonrljr/sysidentpy#107]
Currently, SysIdentPy has a method to return a friendly representation of the regressors, but this method only works for Polynomial Basis Function.

To address this limitation, the maintainer of SysIdentPy (wilsonrljr)  has set a goal to create a new method that can represent different basis functions, such as Fourier and RBF. This new method will provide users with a more versatile way to visualize and interpret the regressors used in their system identification models.

Creating a new method to represent different basis functions will require some work, including developing new algorithms and methods to support this feature. The SysIdentPy maintainer, wilsonrljr, is committed to helping in all steps of the implementation to make the new method available for the users.

The new method will be designed to provide users with a clear and intuitive representation of the regressors, regardless of the basis function used in the model. For example, if Fourier Basis Function is used, the new method will return a representation of the regressors indicating which are related to Fourier function. Similarly, if RBF is used, the new method will return a representation of the radial basis functions used in the model, providing users with a more detailed understanding of the basis function.

The new method will be designed to be easy to use and flexible enough to handle different types of basis functions. It will also be integrated into the existing SysIdentPy framework, making it simple for users to switch between different basis functions and use the new method to analyze their models.

In conclusion, the development of a new method to represent different basis functions in SysIdentPy is an important step forward for the library, and will provide users with a more versatile way to visualize and interpret the regressors used in their system identification models. With the commitment of the SysIdentPy maintainer to help in all steps of the implementation, users can expect to have access to this new feature in the near future.

**Comment by lsamuel41196** [comment]
"Currently, SysIdentPy has a method to return a friendly representation of the regressors, but this method only works for Polynomial Basis Function."

@wilsonrljr where does this method reside? i was unable to find it

**Comment by wilsonrljr** [comment]
Hey @lsamuel41196 , you can find it here: https://github.com/wilsonrljr/sysidentpy/blob/main/sysidentpy/utils/display_results.py
